.github/workflows/indexer_formatting.yml

```diff
@@ -0,0 +1,22 @@
+---
+name: Format indexer
+
+on:
+  workflow_call:
+
+jobs:
+  deno-formatting:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          sparse-checkout: indexer
+      - name: Install zip
+        run: sudo apt update && sudo apt upgrade && sudo apt install zip
```

Comment:
why is this needed?

Reply:
Deno installation requires the `unzip` command which comes with the `zip` package. `zip` isn't installed by default on the image of the CI